### ISSUE TITLE
We should provide the hazard classification in the aggregation report

### ISSUE THREAD

**Issue by samnawi** [AIFDR/inasafe#2455]
I and @fredychandra still doing facility in sub-national level to create the Risk Assessment document. This document is one of mandate from DM Regulation in Indonesia (UU No. 24/2007).
We implement InaSAFE in this process, to get the valuation of each disaster risk.

We found some lack in the report, when we use InaSAFE in this activity.
When we use aggregate, apparently the hazard classification is not include in the detail information. the hazard classification just appear in the general information.
![inasafe1](https://cloud.githubusercontent.com/assets/8871958/10936391/fc2d2212-831e-11e5-9d71-aa53d6e46404.jpg)
![inasafe2](https://cloud.githubusercontent.com/assets/8871958/10936392/fc363712-831e-11e5-8ebf-73fa699e5ca7.jpg)

If we can add the hazard classification in the detail classification, InaSAFE will more benefit and result more useful report. Because the break down information from the number of exposure impacted, is needed to create the Risk Assessment document. So, the user will not doing twice, to calculate the number of impacted (for every class) in province / district level, but also in the same report, they will get the information about the number of exposure impacted in every class at subdistrict / village level.

# Proposed Solution
Could we add one table in the detail information, to give the information about the impacted in every village/subdistrict that also include the hazard classification

Maybe like in this picture
![inasafe3](https://cloud.githubusercontent.com/assets/8871958/10937104/5de88d4e-8323-11e5-9fb5-7ef006e6be8a.jpg)

# CC 
@Charlotte-Morgan @timlinux @ismailsunni 

Thank You

**Comment by Charlotte-Morgan** [comment]
Hi Iwan

What IF are you using. Is it the generic IF?

Charlotte Morgan
Spatial Data Analyst
DMInnovation - Jakarta
GEOSCIENCE AUSTRALIA

Mobile: +62 811 820 4648

On 4 Nov 2015, at 6:40 pm, Adi Kurniawan <notifications@github.com<mailto:notifications@github.com>> wrote:


I and @fredychandra<https://github.com/fredychandra> still doing facility in sub-national level to create the Risk Assessment document. This document is one of mandate from DM Regulation in Indonesia (UU No. 24/2007).
We implement InaSAFE in this process, to get the valuation of each disaster risk.

We found some lack in the report, when we use InaSAFE in this activity.
When we use aggregate, apparently the hazard classification is not include in the detail information. the hazard classification just appear in the general information.
[inasafe1]<https://cloud.githubusercontent.com/assets/8871958/10936391/fc2d2212-831e-11e5-9d71-aa53d6e46404.jpg>
[inasafe2]<https://cloud.githubusercontent.com/assets/8871958/10936392/fc363712-831e-11e5-8ebf-73fa699e5ca7.jpg>

If we can add the hazard classification in the detail classification, InaSAFE will more benefit and result more useful report. Because the break down information from the number of exposure impacted, is needed to create the Risk Assessment document. So, the user will not doing twice, to calculate the number of impacted (for every class) in province / district level, but also in the same report, they will get the information about the number of exposure impacted in every class at subdistrict / village level.

Proposed Solution

Could we add one table in the detail information, to give the information about the impacted in every village/subdistrict that also include the hazard classification

Maybe like in this picture
[inasafe3]<https://cloud.githubusercontent.com/assets/8871958/10937104/5de88d4e-8323-11e5-9fb5-7ef006e6be8a.jpg>

CC

@Charlotte-Morgan<https://github.com/Charlotte-Morgan> @timlinux<https://github.com/timlinux> @ismailsunni<https://github.com/ismailsunni>

Thank You

-
Reply to this email directly or view it on GitHub<https://github.com/AIFDR/inasafe/issues/2455>.

Geoscience Australia Disclaimer: This e-mail (and files transmitted with it) is intended only for the person or entity to which it is addressed. If you are not the intended recipient, then you have received this e-mail by mistake and any use, dissemination, forwarding, printing or copying of this e-mail and its file attachments is prohibited. The security of emails transmitted cannot be guaranteed; by forwarding or replying to this email, you acknowledge and accept these risks.
-------------------------------------------------------------------------------------------------------------------------

**Comment by timlinux** [comment]
@samnawi what do you think about the removal of the school etc. classes - should we simply remove them?

**Comment by samnawi** [comment]
@Charlotte-Morgan no, it's not only occured in generic IF, but also for specific IF like flood, volcano, EQ.
@timlinux I think we still need for the information that you mean, but maybe we can add one table again to give the breakdown information for the number of exposures based on the hazard classification in each area like in this picture
![image](https://cloud.githubusercontent.com/assets/8871958/11268643/67ec9aac-8ee4-11e5-95b7-da0e54056f20.png)

**Comment by timlinux** [comment]
Discussing this again here with @samnawi - he is proposing to not show exposure class columns and rather show hazard level counts in the report.

**Comment by Gustry** [comment]
I think we can work on that for the 4.1 release. It something we can provide with the 4.0 architecture. We talked a little bit in Jakarta to provide one table per aggregation area too. We need to check that together.

**Comment by Charlotte-Morgan** [comment]
hi @felix-yew  can you please see if this is still an issue or maybe already fixed in our standard reports

**Comment by Charlotte-Morgan** [comment]
@ivanbusthomi - can you please check if this is still an issue?

**Comment by Charlotte-Morgan** [comment]
Hi @samnawi 
You can see that I have asked Minions to test this issue and confirm if this is still a requirement  ...
Let me give it back to you. If you want to make this happen; you will need to give specific advice on which fields you need written from the analysis to the rich data to make this table - ie are the fields you want in the rich data already or not??
cc @timlinux @thamrinf @ivanbusthomi @felix-yew

**Comment by samnawi** [comment]
Hi @Charlotte-Morgan what I proposed in this ticket is something that @ivanbusthomi create in https://github.com/inasafe/inasafe/issues/4070#issuecomment-300373155.

Currently, we can see the information which generated by InaSAFE including breakdown by age, gender, vulnerability (age, gender, disability), and minimum needs.

Whether we still have a possibility to add a new group just for total population by hazard like a proposed from @ivanbusthomi in https://github.com/inasafe/inasafe/issues/4070#issuecomment-300373155 ?

This is what I do using Group Stats
![image](https://user-images.githubusercontent.com/8871958/35905622-64bea50e-0c1a-11e8-8268-9b82f353cf2b.png)


For further discussion about rich reporting, should we still keep this separate ticket or we can discuss just in one ticket?
I think this ticket just for hazard classification in the aggregation report, but in #4070 not only about hazard classification.
